### DISCUSSION TITLE
Status bar on a specific monitor

### DISCUSSION THREAD

**Discussion by rekfuki** [glzr-io/glazewm#285] [Q&A]
Is there a way to configure the status bar so that it runs only on a specific monitor?

 I'm using two monitors and I would like to have the status bar only on my secondary monitor.

**Comment by HolbyFPV** [comment]
There is a way to configure different bars for different monitors.  

But as far as removing the bar, how would you expect to see your current workspace or tiling direction on a monitor that has no bar?  The user would be forced to memorize this as they use the WM?  Help me understand your workflow and there is probably a solution for you.

**Reply by rekfuki** [reply]
My `polybar` on linux runs on a secondary monitor and it simply highlights two of the active workspaces that are on each monitor; however, it does not indicate which workspace is open on which monitor. 

For me, typically, the same workspaces are always on the same monitors i.e workspaces 8 and 9  are on the secondary monitor and workspaces 1-7 & 10 are on the primary - this basically never changes.

I also have no indicator for the tiling direction as am I running i3 with always vertical splits. 

I would like to get rid of the bar simply because I've switched to an OLED monitor and static elements like status bars are not great for burn-in reasons.
